sympy/utilities/lambdify.py

```diff
@@ -76,6 +76,7 @@
     "Ci": "ci",
     "RisingFactorial": "rf",
     "FallingFactorial": "ff",
+    "lgamma":"loggamma",
```

Comment:
Are you sure you don't have this backwards?

```suggestion
    "loggamma": "lgamma",
```

Reply:
But then `assert mat in mpmath.__dict__` this test case would fail as `mpmath` also not contains `lgamma`. Now another change would like be `"loggamma": "loggamma"` but this doesn't make any sense according to me. The main point over here is that for `lgamma` the function of `loggamma` should be called that is available in `mpmath.__dict__` and thats exactly what I have done.

Reply:
What is `lgamma`? If it's not a sympy function, why is it on this dict?

Reply:
Because in the related issue it is showing the import error for `lgamma` and It should import `loggamma` when it is showing that error message thats the only reason for adding `lgamma` in the dict. Further computation can be done by `loggamma` itself.

Reply:
Any suggested change?

Reply:
```suggestion
    "loggamma":"loggamma",
```
Assuming that also works. It results in a simpler test, and is what a maintainer suggested in the original issue.

Reply:
Ok, I am on it.

Reply:
If I try 
```
from sympy import *
import mpmath

x = symbols('x')
print(lambdify(x, loggamma(x), 'mpmath'))
```
with `"loggamma": "loggamma"` I still got this error:



```
Traceback (most recent call last):
  File "E:\sympy\jupyter_files\sympy\sympy\utilities\lambdify.py", line 835, in lambdify
    exec_(ln, {}, namespace)
  File "<string>", line 1, in <module>
ImportError: cannot import name 'lgamma' from 'mpmath' (E:\sympy\jupyter_files\test\lib\site-packages\mpmath\__init__.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\sympy\jupyter_files\sympy\sympy\utilities\lambdify.py", line 841, in lambdify
    exec_(ln, {}, namespace)
  File "<string>", line 1, in <module>
NameError: name 'mpmath' is not defined
```
But when `"lgamma": "loggamma"` I got the generated function like this:-
```
<function _lambdifygenerated at 0x00000184C93077B8>
```

Reply:
In `mpmath __init__.py ` if I add `lgamma = mp.loggamma` then also it works and showing the correct output.

Reply:
Thanks for trying that out. Something weird is going on here.

Reply:
> Something weird is going on here.

Absolutely I am also wondering that why its tries to import `lgamma`.

Reply:
My guess is that somewhere a translation from `sympy.loggamma` to  `math.lgamma` happens, and then your patch reverses this mistake.

Reply:
Ah, found it: these lines are incorrect: https://github.com/sympy/sympy/blob/70f83bd004e35fc3b96a1e07699a3db33a84002c/sympy/printing/pycode.py#L499

Reply:
> My guess is that somewhere a translation from `sympy.loggamma` to `math.lgamma` happens, and then your patch reverses this mistake.

Yup